benchmark/src/Utils.php

```diff
@@ -16,7 +16,7 @@ final class Utils
 
     public static function getClient(): Client
     {
-        return self::$client ??= new Client(self::getUri());
+        return self::$client ??= new Client(self::getUri(), ['disableClientPersistence' => true]);
```

Comment:
```suggestion
        return self::$client ??= new Client(self::getUri(), [], ['disableClientPersistence' => true]);
```

`disableClientPersistence` is a driver option (third param).